tests/modules/hmmer/hmmbuild/test.yml

```diff
@@ -0,0 +1,10 @@
+- name: hmmer hmmbuild test_hmmer_hmmbuild
+  command: nextflow run ./tests/modules/hmmer/hmmbuild -entry test_hmmer_hmmbuild -c ./tests/config/nextflow.config  -c ./tests/modules/hmmer/hmmbuild/nextflow.config
+  tags:
+    - hmmer
+    - hmmer/hmmbuild
+  files:
+    - path: output/hmmer/PF14720.hmm.gz
```

Comment:
You can still check for strings in gzipped files with pytest workflow, if you can find a good stirng in there!

Reply:
Sophisticated! Thanks for the tip!